### PR TITLE
Risk level indicator styling

### LINKED ISSUE #1783: Implement new severity label design

### LINKED ISSUE #1827: Object list table doesn't use the whole width
**Describe the bug**
Object list table doesn't use the whole width anymore, see screenshot

**To Reproduce**
Go to the object list. It's visible most of the time, but if there are objects with large names you might need to filter on for example dnszone to clearly see the problem. 

**Screenshots**
image

**OpenKAT version**
Main, 1.12 isn't affected.

### PR DESCRIPTION
**Pull request by HeleenSG**
### Changes
Risk level indicator styling on the new design. 

### Issue link
Closes #1783 and closes #1827

### Proof
Before:
Screenshot 2023-09-20 at 16 28 38

After:
Screenshot 2023-09-20 at 16 28 07

### Code Checklist
- [ ] All the commits in this PR are properly PGP-signed and verified;
- [ ] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: rocky/assets/css/themes/soft/manon/critical-informational-labels.scss [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+// table .low,
+// table .informational,
+// table .recommendation {
+//   width: 100%;
+//   box-sizing: border-box;
+// }
+
+// table .medium {
+//   border: 1px dashed black;
+// }
```
Comment (dekkers) [inline rocky/assets/css/themes/soft/manon/critical-informational-labels.scss:23]: The file doesn't have any content anymore that is not commented out and isn't included anymore as far as I can see, so should't it just be deleted?

**Comment by Darwinkel** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [ ] I have managed to pass the onboarding flow
    - [ ] Objects and Findings are created properly
    - [ ] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* New layout looks good, doesn't appear to cause any regressions

File: rocky/assets/css/themes/soft/manon/critical-informational-labels.scss (hunk 1 above)
Comment (HeleenSG) [reply rocky/assets/css/themes/soft/manon/critical-informational-labels.scss]: Ah yes. Good catch I've removed it. Thanks

**Comment by HeleenSG** [comment]
> ## Checklist for QA:
> * [x]  I have checked out this branch, and successfully ran a fresh `make reset`.
> * [x]  I confirmed that there are no unintended functional regressions in this branch:
>   
>   * [ ]  I have managed to pass the onboarding flow
>   * [ ]  Objects and Findings are created properly
>   * [ ]  Tasks are created and completed properly
> * [x]  I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
> 
> ### What works:
> * New layout looks good, doesn't appear to cause any regressions
> 
> ### What doesn't work:
> * Minor spacing issue when look at findings for a specific object
>   image

Good find! Thanks for letting me know. Latest commit should solve this issue.

**Comment by TwistMeister** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

Screenshot 2023-09-25 at 14 02 44
Screenshot 2023-09-25 at 14 03 04
Screenshot 2023-09-25 at 14 03 53

**Review by TwistMeister — APPROVED** [review]

**Review by dekkers — APPROVED** [review]

**Pull request merged**